easybuild/easyconfigs/l/libFLAME/libFLAME-5.2.0-GCC-10.2.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'ConfigureMake'
+
+name = 'libFLAME'
+version = '5.2.0'
+
+homepage = 'https://developer.amd.com/amd-cpu-libraries/blas-library/#libflame'
+description = """libFLAME is a portable library for dense matrix computations,
+providing much of the functionality present in LAPACK."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/flame/libflame/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['997c860f351a5c7aaed8deec00f502167599288fd0559c92d5bfd77d0b4d475c']
+
+# '--enable-max-arg-list-hack --enable-dynamic-build' requires 'file' function from GNU Make 4.x
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Python', '3.8.6'),
+    ('make', '4.3'),  # needed on Cent OS 7  where make 3 is installed
+]
+
+dependencies = [('BLIS', '0.8.0')]
+
+# Use unset FLIBS to let configure pick up LDFLAGS
+preconfigopts = 'unset FLIBS && '
+preconfigopts += 'LIBS="-lblis $LIBS" '
+preconfigopts += 'LDFLAGS="$LDFLAGS -L$EBROOTBLIS/lib -fopenmp -lm -lpthread" '
+preconfigopts += 'CFLAGS="$CFLAGS -I$EBROOTBLIS/include/blis" '
+
+configopts = '--enable-max-arg-list-hack '
+configopts += '--enable-lapack2flame '
+configopts += '--enable-external-lapack-interfaces '
+configopts += '--enable-cblas-interfaces '
+configopts += '--enable-dynamic-build '
+configopts += '--enable-multithreading=openmp '
+
+# libFLAME C++ Template API tests
+# runtest = 'checkcpp LIBBLAS=$EBROOTBLIS/lib/libblis.a'
+
+sanity_check_commands = [
+    'cd %(builddir)s/%(namelower)s-%(version)s/test '
+    '&& mkdir obj '
+    '&& make LIBBLAS=$EBROOTBLIS/lib/libblis.so LDFLAGS="-fopenmp -lm -lpthread" HOST="x86_64-pc-linux-gnu" LIBLAPACK="" '
```

Comment:
Hmm, we shouldn't use `x86_64-pc-linux-gnu` hardcoded here.

Ideally we use the updated `config.guess` that EasyBuild uses when using the `ConfigureMake` easyblock, but we can't really do that easily from an easyconfig file.

Since this easyconfig is becoming a bit complex, perhaps an easyblock for `libFLAME` is a better way forward?

Reply:
Yes, I agree an easyblock makes more sense.

I will close this PR and open another one once I had time to create the easyblock.